### PR TITLE
Use new thread labs feature name

### PR DESCRIPTION
**Pull request by gsouquet**
## Checklist

-   [ ] ~~Tests written for new code (and old code if feasible)~~
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Use new thread labs feature name ([\#9748](https://github.com/matrix-org/matrix-react-sdk/pull/9748)).

### CODE REVIEW

**Review by andybalaam — APPROVED** [review]

**Review by t3chguy — APPROVED** [review]

File: src/Unread.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export function doesRoomHaveUnreadMessages(room: Room): boolean {
     // despite the name of the method :((
     const readUpToId = room.getEventReadUpTo(myUserId!);
 
-    if (!SettingsStore.getValue("feature_thread")) {
+    if (!SettingsStore.getValue("feature_threadstable")) {
```
Comment (turt2live) [inline src/Unread.ts:69]: please also update labs/betas.md in element-web :)

Comment (gsouquet) [reply src/Unread.ts:69]: Not as part of this PR as develop.element.io is broken

Comment (t3chguy) [reply src/Unread.ts:69]: That file is in a different repo so can't be part of this PR anyway

Comment (turt2live) [reply src/Unread.ts:69]: yup, just mentioning it as a step to complete

**Comment by gsouquet** [comment]
@matrix-org/element-web-app-team the `i18n` check fails, but running `yarn i18n` does not yield anything locally.

**Comment by turt2live** [comment]
git is super angry about this for some reason on the i18n check - for safety, I've remade the PR at https://github.com/matrix-org/matrix-react-sdk/pull/9749 and will merge that one (considering approval can't be transferred easily)

**Pull request closed without merging**